automated_packaging/update_cloudformation.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+use lib '/usr/local/bin';
+use common_functions;
+
+$NEW_VERSION=$ARGV[0];
+$OLD_VERSION=$ARGV[1];
+
+`git checkout master`;
+
+# Update configuration file
+$release_major = substr($OLD_VERSION, 0, 1);
+`sed -i -- 's/$OLD_VERSION/$NEW_VERSION/g' citus-$release_major/citusdb.json`;
```

Comment:
If we are releasing major version, we need to create a new directory and copy files from old version's directory.